cli/src/pcluster/validators/fsx.py

```diff
@@ -34,3 +69,131 @@ def __call__(self, import_path, imported_file_chunk_size, export_path, auto_impo
                 "When specifying auto import policy, the import path option must be specified",
                 FailureLevel.CRITICAL,
             )
+
+    def _validate_persistent_options(self, deployment_type, kms_key_id, per_unit_storage_throughput):
+        if deployment_type == "PERSISTENT_1":
+            if not per_unit_storage_throughput:
+                self._add_failure(
+                    "'per_unit_storage_throughput' must be specified when 'deployment_type = PERSISTENT_1'",
```

Comment:
let's rememeber to cleanup all the validation messages. We can do it in a separate PR.